packages/encrypt/Cryptor.js

```diff
@@ -36,8 +36,15 @@ class Cryptor {
         }
 
         const { AES_KEY, AES_KEY_ID } = process.env;
+        console.log('JON >>> process.env.AES_KEY', process.env.AES_KEY);
         const randomKey = crypto.randomBytes(32).toString('hex').slice(0, 32);
 
+        if (!AES_KEY || !AES_KEY_ID) {
```

Comment:
I _think_ this needs to be `if (!AES_KEY && !AES_KEY_ID)` as we're trying to say "use the key and key ID for manual key rotation"